lib/workers/repository/process/index.ts

```diff
@@ -103,9 +104,11 @@ export async function extractDependencies(
     logger.debug('No baseBranches');
     const packageFiles = await extract(config);
     addSplit('extract');
-    res = await lookup(config, packageFiles);
+    if (GlobalConfig.get('dryRun') !== 'extract') {
+      res = await lookup(config, packageFiles);
+      addSplit('lookup');
+    }
   }
-  addSplit('lookup');
   return res;
```

Comment:
I'd prefer to revert these changes and instead add logic in the right place for "if dryRun===extract then return"